scripts/ci_steps/codegen_sdk.sh

```diff
@@ -12,6 +12,7 @@ rm -rf Tests/Services/*
 ./gradlew --stop
 
 # Regenerate the SDK Package.swift with all services
+unset AWS_SWIFT_SDK_USE_LOCAL_DEPS
```

Comment:
The CLI now uses the latest stable aws-sdk-swift as a dependency (to perform the S3 & CloudFront operations needed to publish.)

If you leave that env var set, it will cause this dependency to break, because it will try to use a local copy of smithy-swift that isn't there.

For context, I expect to be removing the `AWS_SWIFT_SDK_USE_LOCAL_DEPS` env var entirely before GA.  We'll use a different mechanism for local dependencies during development.